packages/node_modules/@ciscospark/plugin-messages/src/messages.js

```diff
@@ -3,7 +3,13 @@
  */
 
 import {
+  WEBEX_EVENT,
```

Comment:
For the next "envelope" PR, we may want to consolidate these into a parent so they're easier to import.